src/components/PeopleTable.tsx

```diff
@@ -1,682 +1,73 @@
-export const PeopleTable = () => {
+import classNames from 'classnames';
+
+import { Person } from '../types';
+import { OrderOption, SortOptions } from '../utils/data';
+import { PersonRow } from './PersonRow';
+import { SearchLink } from './SearchLink';
+
+type Props = {
+  people: Person[];
+  sort: SortOptions;
+  order: OrderOption;
+};
+
+export const PeopleTable: React.FC<Props> = ({
+  people,
+  sort,
```

Comment:
The 'sort' and 'order' from props should have default values in case they are not provided.